lib/Traits/CsvCatalogItemTraits.ts

```diff
@@ -33,6 +34,7 @@ class PollingTraits extends ModelTraits {
 }
 
 export default class CsvCatalogItemTraits extends mixTraits(
+  AutoRefreshingTraits,
```

Comment:
Maybe this shouldn't be here, as all `AutoRefreshingTraits` are computed values in `CsvCatalogItem`